app/views/participants/show.html.haml

```diff
@@ -0,0 +1,16 @@
+:ruby
+- set_meta_tags title: @participant.name
```

Comment:
This `:ruby` filter is unnecessary.
Can you use the one of the followings:

``` haml
:ruby
  set_meta_tags title: @participant.name
```

or 
``` haml
- set_meta_tags title: @participant.name
```